docs/core/plugins/quota.md

```diff
@@ -5,19 +5,28 @@ dovecotlinks:
   quota_admin:
     hash: quota-admin-commands
     text: quota admin
+  quota_limits:
+    hash: quota-limits
+    text: quota limits
+  quota_backends:
+    hash: quota-backends
+    text: quota backends
```

Comment:
Same comment about the titlecase as above. This is a bit unfortunate, as there are existing links that are already lowercased and I don't think you should change them in this here commit, but if you feel inclined: Change the existing titles to be titlecased as well, in another commit, though.

Reply:
I think we don't need to be so strict about the commits in documentation. We're not likely going to be digging back much in documentation history (while with code it's rather common).